manifests/nfd-daemonset-master-worker.yaml.template

```diff
@@ -102,7 +102,7 @@ spec:
               mountPath: "/host-etc/os-release"
               readOnly: true
             - name: host-sys
-              mountPath: "/host-sys"
+              mountPath: "/host/sys"
```

Comment:
Why is this change needed?
It is working as it is right now
see: https://github.com/kubernetes-sigs/node-feature-discovery/search?q=host-sys

Reply:
This (and all the related changes) is driven by the needs of the package we use to learn about the hardware details (ghw: https://github.com/jaypipes/ghw). In this case, `ghw` can't really read data from sysfs from a random directory, while supports just fine chroot: https://github.com/jaypipes/ghw/blob/v0.8.0/README.md#overriding-the-root-mountpoint-ghw-uses

If this rename is a issue we can find another solution.

Reply:
Another approach: https://github.com/jaypipes/ghw/issues/249

Reply:
I think this is a welcome change. This `host-*` business just started with one mount and has been breeding uncontrollably 😄